recurring_contract/models/contract_group.py

```diff
@@ -304,3 +286,25 @@ def _setup_inv_data(self, journal, invoicer, contracts):
             'narration': "\n".join(contracts.mapped(lambda c: c.comment or ""))
         }
         return inv_data
+
+    def _updt_invoices_cg(self, vals):
+        """ method to update invoices on contrat group (cg)
+            :params vals dict of value that has been modified on the cg
+        """
+        self.ensure_one()
+        if "ref" in vals or "payment_mode_id" in vals or "advance_biling_months" in vals:
```

Comment:
`advance_biling_months` is tested but not used at all in the update. I think it has no impact on the change of the invoices and could be removed from the test here, what do you think?